modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/playback/SleepTimer.kt

```diff
@@ -16,10 +18,19 @@ import javax.inject.Singleton
 import au.com.shiftyjelly.pocketcasts.localization.R as LR
 
 @Singleton
-class SleepTimer @Inject constructor(@ApplicationContext private val context: Context) {
+class SleepTimer @Inject constructor(
+    private val analyticsTracker: AnalyticsTrackerWrapper,
+    @ApplicationContext private val context: Context,
+) {
 
     companion object {
         private var sleepTimeMs: Long? = null
+        private var lastSleepAfterTimeInMinutes: Int = 0
+        private var lastTimeHasFinishedInMillis: Long = 0
```

Comment:
Instead of juggling conversions between milliseconds and minutes which is very error prone could you change these properties to hold their values as `Duration` type?